src/playground/App.js

```diff
@@ -49,9 +50,39 @@ const fillOptionsDefaults = options =>
             }, {})
         });
 
+const convertLegacyOptionsToFlatConfig = (options = {}) => {
+
+    // If options object is empty, return it
+    if (options && Object.keys(options).length === 0 && options.constructor === Object) {
+        return options;
+    }
+
+    // If options is already a flat config, return it
+    if (typeof options.languageOptions !== "undefined" && typeof options.parserOptions === "undefined") {
+        return options;
+    }
+
+    const { parserOptions } = options;
+    const flatConfigOptions = {
+        languageOptions: {
+            ecmaVersion: parserOptions.ecmaVersion || "latest",
+            parserOptions: {
+                ecmaFeatures: parserOptions.ecmaFeatures || {}
+            },
+            sourceType: parserOptions.sourceType || "module"
+        },
+        rules: options.rules
```

Comment:
Links from the docs site don't work. For example, this is with `#...` copied from the first example for the [`accessor-pairs`](https://eslint.org/docs/latest/rules/accessor-pairs#setwithoutget) rule:

https://deploy-preview-509--new-eslint.netlify.app/play/#eyJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsic291cmNlVHlwZSI6Im1vZHVsZSJ9fSwidGV4dCI6Ii8qZXNsaW50IGFjY2Vzc29yLXBhaXJzOiBcImVycm9yXCIqL1xuXG52YXIgbyA9IHtcbiAgICBzZXQgYSh2YWx1ZSkge1xuICAgICAgICB0aGlzLnZhbCA9IHZhbHVlO1xuICAgIH1cbn07XG5cblxudmFyIG8gPSB7ZDogMX07XG5PYmplY3QuZGVmaW5lUHJvcGVydHkobywgJ2MnLCB7XG4gICAgc2V0OiBmdW5jdGlvbih2YWx1ZSkge1xuICAgICAgICB0aGlzLnZhbCA9IHZhbHVlO1xuICAgIH1cbn0pOyJ9

```
playground.js:2 TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at rv (playground.js:2:2203255)
    at wo (playground.js:2:131194)
    at Cl (playground.js:2:188149)
    at bc (playground.js:2:177131)
    at Ec (playground.js:2:177059)
    at yc (playground.js:2:176922)
    at oc (playground.js:2:173729)
    at sc (playground.js:2:172284)
    at C (playground.js:2:279135)
```

When I try this locally (with the debug info), it seems that the error is thrown when trying to do `Object.keys(options.rules)`.

I _think_ this happens because this function always creates `rules` property, even if the value is `undefined`, so in `fillOptionsDefaults` it overwrites `rules: {}`.

Reply:
I think this broke after I rebased on top of https://github.com/eslint/eslint.org/commit/44cfc74d2de96c9bcaa6c143b1a372fc938abb5f, I'll push a fix.